corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -264,3 +265,28 @@ def delete_diff_db(domain):
         # reraise on any other error
         if e.errno != 2:
             raise
+
+
+BASE_IGNORED_FORM_PATHS = {
+    '_rev',
+    'migrating_blobs_from_couch',
+    '#export_tag',
+    'computed_',
+    'state',
+    'edited_on',
+    'computed_modified_on_',
+    'problem',
+    'orig_id',
+    'deprecated_form_id',
```

Comment:
for some of these we should still find a way to compare the couch and sql versions if we're not already